scipy/special/_legacy.pxd

```diff
@@ -135,8 +135,20 @@ cdef inline double smirnov_unsafe(double n, double e) nogil:
     _legacy_cast_check("smirnov", n, 0)
     return smirnov(<int>n, e)
 
+cdef inline double smirnovc_unsafe(double n, double e) nogil:
```

Comment:
No new functions should be added to `_legacy.pxd`; these exist only for back-compatibility reasons.

Reply:
Ah never mind, being foolish.

Reply:
When adding C code, my approach is a combination of reactive (fix what fails), and proactive (grep smirnov *.*; do same for smirnovc).  The changes to _legacy.pxd come from the latter.  Are you saying that I can ignore _legacy.pxd from now on when adding new functions to special?

Reply:
Yes-the story is that in the past some Cephes functions accepted doubles and silently cast them to ints. The signatures were changed to fix this, but to preserve back-compatibility these wrappers were added which preserved the old behavior but added a warning. We don't want any new functions to have the implicit rounding, however.